frontend/src/modules/machinery/components/CaighdeanTranslation.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CaighdeanTranslation from './CaighdeanTranslation';
+
+
+describe('<CaighdeanTranslation>', () => {
+    it('renders the component properly', () => {
+        const source = {
+            type: 'caighdean',
+            url: 'https://github.com/kscanne/caighdean',
+            id: 'machinery-CaighdeanTranslation--visit-caighdean',
+            string: 'Caighdean',
+            title: 'Visit Caighdean Machine Translation',
```

Comment:
Only the `type` value is needed here. The rest is not used in the component (except `url` but I think it should be removed).